src/css/screens/watchcard.css

```diff
@@ -0,0 +1,46 @@
+.watchcard{
+    margin-bottom: 40px;
+    width: 270px;
+    position: relative;
+}
+.watchcard-thumbnail{
+    height: 140px;
+    width: 250px;
+}
+.watchcard-thumbnail:hover{
+    transform: scale(1.03);
+    cursor: pointer;
+}
+.watchcard-info{
+    display: flex;
+    padding-top: 10px;
+}
+.watchcard-text{
+    margin-left: 10px;
+}
+.watchcard-text h4{
+    margin-bottom: 5px;
+    font-size: var(--normal-font-size);
+}
+.watchcard-text p{
+    color: var(--text-color-light);
+    font-size: var(--small-font-size);
+}
+.watchcard-avatar{
+    height: 30px !important;
```

Comment:
try to avoid using important by increasing specificity

Reply:
sure